docs/pages/2020_Intro_Week/sections/running.rst

```diff
@@ -589,16 +589,70 @@ Finally, let's label this calculation as a restarted one and submit the new calc
 Inspect the restarted calculation to verify that, this time, it completes successfully.
 You should see a "finished" status with exit code zero when running ``verdi process list -a -p1``.
 
+Calculation results
+===================
 
-Using the calculation results
-=============================
+The results of a calculation can be accessed directly from the calculation node using the following:
 
-Now you can access the results as you have seen earlier. For example, note down the PK of the calculation so that you can load it in the ``verdi shell`` and check the total energy with the commands:
+.. code-block:: console
 
-.. code:: python
+    $ verdi calcjob res <IDENTIFIER>
+
+which will print the output dictionary of the 'scalar' results parsed by AiiDA at the end of the calculation.
+Note that this is actually a shortcut for:
+
+.. code-block:: console
+
+    $ verdi data dict show <IDENTIFIER>
+
+where ``IDENTIFIER`` refers to the ``Dict`` node attached as an output of the calculation node, with link name ``output_parameters``.
+By looking at the output of the command, what is the Fermi energy of the calculation you have run?
+
+Similarly to what you did for the calculation inputs, you can access the output files via the commands:
+
+.. code-block:: console
+
+    $ verdi calcjob outputls <IDENTIFIER>
+
+and
+
+.. code-block:: console
+
+    $ verdi calcjob outputcat <IDENTIFIER>
+
+Use the latter to verify that the Fermi energy that you have found in the last step has been extracted correctly from the output file.
+
+.. tip::
 
-    calculation = load_node(<pk>)
-    calculation.res.energy
+    Filter the lines containing the string 'Fermi', e.g. using ``grep``, to isolate the relevant lines.
+
+The results of calculations are stored in two ways: ``Dict`` objects are stored in the database, which makes querying them very convenient, whereas ``ArrayData`` objects are stored on the disk.
+Once more, use the command ``verdi data array show <IDENTIFIER>`` to determine the Fermi energy obtained from calculation you ran.
+This time you will need to use the identifier of the output ``ArrayData`` of the calculation, with link name ``output_trajectory_array``.
```

Comment:
Ok, so I think the `ArrayData` called `output_trajectory_array` is now a more specific `TrajectoryData` called `output_trajectory`. Also, the Fermi energy asked here is just what is reported as the "energy", right?

Reply:
> Ok, so I think the `ArrayData` called `output_trajectory_array` is now a more specific `TrajectoryData` called `output_trajectory`. 

Yes. This was also mentioned by @sphuber in #236. Unfortunately, we run an `scf` calculation here, so the `TrajectoryData` is not very interesting. I'll mention this and have them run a `vc-relax` as a final exercise.

> Also, the Fermi energy asked here is just what is reported as the "energy", right?

The Fermi energy is indeed not present in this calculation. I've adjusted this to just looking for the total energy (`energy`). 

Seems moving this section here caused more problems than I thought... Sorry about that.